build.gradle

```diff
@@ -43,3 +46,10 @@ xtext {
 test {
     useJUnitPlatform()
 }
+
+jar {
```

Comment:
How do you build the JAR and run the program @joaoasrosa? 
Maybe you could add the Gradle "application" plugin which generates distributions including all dependencies and start scripts (sh; and bat for windows) automatically (maybe even easier to start the CLI?).

Reply:
(would be easily done by adding `id "application"` in the `plugins`)